5.0/en/0x05-For-Users-Of-4.0.md

```diff
@@ -1,86 +1,84 @@
-* Web Frontend Security - With a growing number of security considerations when rendering applications in a browser as well as an increase in API only development where these considerations would not be relevant, it was decided to split these requirements into a dedicated chapter.
-* Secure Coding and Architecture - A number of new requirements were developed that related to general security considerations which are required to make the application secure, which did not fit well into existing chapters. We grouped these together into this new chapter.
+* **Self-contained Tokens** – Previously included under session management, the expanded use cases and associated security risks of self-contained tokens warranted a dedicated chapter, with many requirements newly introduced in version 5.x.
+* **Web Frontend Security** – With the increasing complexity of browser-based applications and the rise of API-only architectures, frontend security requirements have been separated into their own chapter.
+* **Secure Coding and Architecture** – New requirements addressing general security practices that did not fit within existing chapters have been grouped here.
 
-There were also some other moves that took effect in version 5.0 to "send a message". Such as, Input Validation was moved away from Sanitization and Encoding to be together with business logic, since the intent of input validation is to enforce the business rules for the input being received.
+Other organizational changes in version 5.0 were made to clarify intent. For example, input validation requirements were moved alongside business logic, reflecting their role in enforcing business rules, rather than being grouped with sanitization and encoding.
 
-The Old V1 Architecture chapter has been removed. The first section of this chapter was primarily requirements which were not in scope and the subsequent sections mapped to other individual chapters. The requirements from these subsequent which were still in scope were therefore moved to the relevant chapters themselves, being deduplicated and clarified along the way.
+The former V1 Architecture chapter has been removed. Its initial section contained requirements that were out of scope, while subsequent sections have been redistributed to relevant chapters, with requirements deduplicated and clarified as necessary.
 
-## Removed direct mappings to other standards
+## Removal of Direct Mappings to Other Standards
 
-We removed the mappings that appear on the face of the standard with the aim of mapping version 5.0 just to the OWASP Common Requirement Enumeration (CRE) project which will map ASVS to a variety of other OWASP projects and external standards.
+Direct mappings to other standards have been removed from the main body of the standard. Version 5.0 now maps primarily to the OWASP Common Requirement Enumeration (CRE) project, which in turn links ASVS to a range of OWASP projects and external standards.
```

Comment:
> Version 5.0 now maps primarily to the

That is not correct. ASVS itself do not have any mapping, the aim and vision is to rely on OpenCRE.

Reply:
Thanks, corrected